onmt/opts.py

```diff
@@ -755,6 +760,9 @@ def translate_opts(parser):
     group.add('--src', '-src', required=True,
               help="Source sequence to decode (one line per "
                    "sequence)")
+    group.add("-src_feats", "--src_feats", required=False,
+              help="Source sequence features (one line per "
```

Comment:
Add an example of the dict format?

Reply:
Maybe we could use `parse_known_args` to dynamically add the features files from `--feat_name1 path/name1.txt --feat_name2 path/name2.txt`. This would be a more user friendly method to set the features files. However, `-h` would not show them...

What do you think?

Reply:
Hmm, not sure that's really more simple. Let's keep the dict string for now, but just add an example of the format in the helpstring.